pkg/addons/registry/static/metadata.yaml

```diff
@@ -12,4 +12,4 @@ images:
         repo: proxy.replicated.com/anonymous/replicated/ec-registry
         tag:
             amd64: 2.8.3-r0@sha256:5b76ebd0a362009e31a05ac487c690f5ece0e11f6c4d9261ca63a3f162b57660
-            arm64: 2.8.3-r0@sha256:b6b7345a1455ae1decd79f39e9ee9f8dff4dedf9fc52acebeba0903779c37cc3
+            arm64: 2.8.3-r0-arm64@sha256:e9841478ded46f9ed3f258ea7b291718954cecce2d6907df778d223dded2dd9d
```

Comment:
hmm, the arm image isn't multi-arch, why did the digest change here?

Reply:
<img width="755" alt="Screenshot 2024-09-17 at 1 16 22 PM" src="https://github.com/user-attachments/assets/f20902d2-ed53-4a74-8761-b9291eaae553">

looks like the tag points at e9

Reply:
sounds good